src/test/java/emissary/output/DropOffUtilTest.java

```diff
@@ -467,50 +465,51 @@ public void testGetEventDate() {
 
     @Test
     public void testGetFileType() {
```

Comment:
Recommend including the original test, since you're now deprecating the method instead of removing it.